host/src/components/EnableConfidenceCard.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
card used to enable confidence on the teacher side, prior to starting the question